src/Interpreter.ts

```diff
@@ -144,25 +201,68 @@ export default class Interpreter {
   }
 
   /**
-   * Determines whether there are any templates missing, i.e. the existing
-   * templates do not account for all contests in all ballot styles.
+   * Gets information about missing templates. Note that until we start adding
+   * templates, we will not know how many pages each ballot should be and
+   * therefore won't have perfect knowledge of how many templates are missing.
+   * To account for this, template metadata for which we know nothing about
+   * either the ballot style or the precinct will have a single entry to
+   * represent it and a page count of -1.
    */
-  public hasMissingTemplates(): boolean {
+  public *getMissingTemplates(): Generator<BallotPageMetadata> {
```

Comment:
What is this * ?

Reply:
Generator function. In Python it simply checks whether you use `yield` anywhere. In JS you mark it like so.